js/renovation/ui/box/__tests__/box.test.tsx

```diff
@@ -20,4 +26,68 @@ describe('Box > Attrs', () => {
       expect(classList.contains('dx-box-flex')).toEqual(true);
     });
   });
+
+  each([1, 'text', true, undefined, null]).describe('cssStyles: %o', (style) => {
+    it('Should pass styles to container from cssStyles', () => {
+      const box = mount<Box>(viewFunction({ cssStyles: style } as any));
+      const rootContainer = box.find(Widget);
+
+      expect(rootContainer.props().style).toEqual(style);
+    });
+  });
+
+  describe('cssStyles', () => {
```

Comment:
These tests assign something to a props property and check the result.
Currently, the result is 'cssStyles' only, but there will be more results.
Let's rename it like this: describe('Check result of props values',

Reply:
Yep, fixed